installing/installing-fips.adoc

```diff
@@ -68,11 +68,11 @@ To install a cluster in FIPS mode, follow the instructions to install a customiz
 
 * xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[Amazon Web Services]
 * xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[Microsoft Azure]
-* xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Bare metal]
+* xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal]
```

Comment:
We don't capitalize here?

Reply:
Perhaps since it's standalone, the capital `Bare` is OK. We far and away lowercase `bare metal`, but I do see a few instances, like in lists or tables, where it's still capitalized. I will revert for now.